cli/command/new/templates.go

```diff
@@ -1,74 +1,142 @@
 package new
 
-var configTpl = `apiVersion: v1alpha
-metadata:
-  name: Feature XYZ
-  tags:
-    - e2e
-    - release team
-    - other tag
-  build: Release $CI_COMMIT_SHORT_SHA
-files:
-  - {{ .TestsFolder }}/example.test.js
-suites:
-  - name: "saucy test"
-    match: "{{ .Match }}"
-image:
-  base: {{ .Name }}
-  version: {{ .Version }}
-sauce:
-  region: {{ .Region }}
-`
-
-// SetupTemplate describes a template for a setup
-type SetupTemplate struct {
-	Filename string
-	Code     string
+import (
+	"archive/tar"
+	"bytes"
+	"compress/gzip"
+	"context"
+	"fmt"
+	"github.com/google/go-github/github"
+	"github.com/rs/zerolog/log"
+	"github.com/tj/survey"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+)
+
+var (
+	overWriteAll     = false
+	templateFileName = "saucetpl.tar.gz"
+)
+
+// GetReleaseArtifactURL provides template artifact url for a given repo
+func GetReleaseArtifactURL(org string, repo string) (string, error) {
+	ctx := context.Background()
+	ghClient := github.NewClient(nil)
+	release, _, err := ghClient.Repositories.GetLatestRelease(ctx, org, repo)
+	if err != nil {
+		return "", err
+	}
+
+	for _, asset := range release.Assets {
+		if *asset.Name == templateFileName {
+			return asset.GetBrowserDownloadURL(), nil
+		}
+	}
+	return "", fmt.Errorf("No %s found", templateFileName)
+}
+
+func createFolder(name string, mode int64) error {
```

Comment:
Doesn't provide much value imho. It's just a method that wraps one method call.

Reply:
Removed